stdlib/Random/src/misc.jl

```diff
@@ -220,6 +220,20 @@ function shuffle!(r::AbstractRNG, a::AbstractArray)
     return a
 end
 
+function shuffle!(r::AbstractRNG, a::AbstractArray{Bool})
+    old_count = count(a)
+    x = 2old_count <= length(a)
```

Comment:
I do not have a specific suggestion, but I would love if we came up with a better name for `x`.

Reply:
This is the value (`true` or `false`) that there are at most `length(a)/2` of.

So `uncommon_value` seems accurate. I added `value` because a `Bool` can easily be thought of as a condition and this is more of a value than a condition IMO (even though we branch on it)